### PR TITLE
feat: cs-376 add manual fluent-bit monitoring configs

### PR DESCRIPTION
**Pull request by arthur-observe**
Creates manually deployable set of configs to support health monitoring of fluentbit agent

### CODE REVIEW

**Comment by obs-gh-abhishekrao** [comment]
Why is this a manual process?

**Comment by arthur-observe** [comment]
It's a manual process becasue we haven't made a decision to default it or make it an option during install and that would be another tranche of work.

**Comment by arthur-observe** [comment]
Ran tests manually locally need to move new set of tests in as separate pr

File: other_configs/fluent_monitoring/README.md
Comment (obs-gh-nikhildua) [inline other_configs/fluent_monitoring/README.md:1]: Can we get a high level overview of what "other_configs" in the readme? Also a "why" they are needed section?

**Review by obs-gh-nikhildua — APPROVED** [review]
Just need more info on README. Perhaps even our docs

**Pull request merged**